src/main/kotlin/org/quiltmc/community/database/entities/OwnedThread.kt

```diff
@@ -13,4 +13,5 @@ data class OwnedThread(
 
     val owner: Snowflake,
     val guild: Snowflake,
+    var save: Boolean,
```

Comment:
This isn't descriptive enough IMO - could call it `preventArchiving`.

Also, you can just set a default value of `false` since most of the code using the database won't need to set it at all.